Dockerfile

```diff
@@ -13,13 +18,21 @@ WORKDIR /app
 # Copy package.json and package-lock.json
 COPY package.json ./
 COPY pnpm-lock.yaml ./
+COPY scripts ./scripts
+
+ENV GIT_SHA=${GIT_SHA}
+ENV BUILD_TIME=${BUILD_TIME}
 
 # Install dependencies
 RUN pnpm install --frozen-lockfile
 
 # Copy the rest of the application code
 COPY . .
 
+RUN GIT_SHA="${GIT_SHA:-$(git rev-parse --short=12 HEAD 2>/dev/null || echo unknown)}" \
+    BUILD_TIME="${BUILD_TIME:-$(date -u +%Y-%m-%dT%H:%M:%SZ)}" \
+    pnpm run prebuild
```

Comment:
We don't need to run `prebuild` here since it will run automatically with `pnpm build`: https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts

Reply:
I've [updated](https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/pull/47/commits/dc0231653e59cc67c04d419df73979781868b7b5) the Dockerfile based on your suggestions